charts/argus/templates/_collector_config.tpl

```diff
@@ -0,0 +1,9 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{- define "collector-config" -}}
+replicas: {{ .Values.collector.replicas }}
+size: {{ .Values.collector.size | quote}}
+useEA: {{ .Values.collector.useEA }}
```

Comment:
Please add check for useEA & escalationChainID before assigning.
Reference : https://github.com/logicmonitor/helm-charts-qa/blob/66ae3698b5dd9af72e1996be5d8674fb2ce9b2ae/charts/argus/templates/collectorset.yaml#L31